docs/api-guide/serializers.md

```diff
@@ -774,6 +774,11 @@ Here's an example of how you might choose to implement multiple updates:
 
     class BookSerializer(serializers.Serializer):
         ...
+        id = serializers.IntegerField(
+            read_only=False,
+            required=False
+        )
```

Comment:
Wonderful - congratulations! :)

> Will try to get to this by the end of the week

No matter either way - I'll get around to it at some point otherwise.